### PR TITLE
Feature: Stable rust framework version

### PR DESCRIPTION
**Pull request by huythanhcere**
Migrate PoS Node template build to stable rust framework and up Network with it on DEV. Ticket ([url](https://cerenetwork.atlassian.net/browse/CBI-327))
Build result: [url](https://github.com/Cerebellum-Network/pos-network-node/runs/2163025986?check_suite_focus=true)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -11,7 +11,7 @@ RUN apt-get update && \
 RUN curl https://sh.rustup.rs -sSf | sh -s -- -y && \
          export PATH=$PATH:$HOME/.cargo/bin && \
          scripts/init.sh && \
-         cargo +nightly-2020-10-06 build --$PROFILE
+         cargo build --$PROFILE
```
Comment (AndreiNavoichyk) [inline Dockerfile]: Which runtime version will be used here? It should be explicitly written.

File: scripts/init.sh [hunk 2]
```diff
@@ -9,8 +9,4 @@ if [ -z $CI_PROJECT_NAME ] ; then
    rustup update stable
 fi
 
-rustup target add wasm32-unknown-unknown --toolchain nightly
-rustup component add rust-src --toolchain nightly
-rustup target add wasm32-unknown-unknown --toolchain stable
-rustup install nightly-2020-10-06
-rustup target add wasm32-unknown-unknown --toolchain nightly-2020-10-06
+rustup target add wasm32-unknown-unknown --toolchain nightly
```
Comment (AndreiNavoichyk) [inline scripts/init.sh]: Why do we still have a `nightly` here?

Comment (huythanhcere) [reply scripts/init.sh]: Actually we use nightly latest, stable version while build. Polkadot reference [link](https://github.com/paritytech/polkadot/blob/master/scripts/init.sh). Line 12.

Comment (AndreiNavoichyk) [reply scripts/init.sh]: @huythanhcere 
The latest rust stable version for now is [1.50.0](https://github.com/rust-lang/rust/releases). If we don't have any feature blockers it should be used.

File: Dockerfile (hunk 1 above)
Comment (huythanhcere) [reply Dockerfile]: In init.sh, latest nightly will be used.

File: scripts/init.sh (hunk 2 above)
Comment (huythanhcere) [reply scripts/init.sh]: i removed nightly, checking build. If build successfully, i will commit.

Comment (huythanhcere) [reply scripts/init.sh]: @AndreiNavoichyk if remove nighty, there is error ([url](https://drive.google.com/file/d/1diqqqNnvXURRulXWGZLAt7FGIs3_Syrx/view?usp=sharing))

Comment (huythanhcere) [reply scripts/init.sh]: I think you should approve this PR. There is not any improvement that we can do.

Comment (AndreiNavoichyk) [reply scripts/init.sh]: @huythanhcere 
We should use `stable` toolchain here. Any reasons why not?
`rustup target add wasm32-unknown-unknown --toolchain stable`

Comment (huythanhcere) [reply scripts/init.sh]: Error, details here: https://github.com/Cerebellum-Network/pos-network-node/runs/2191567050?check_suite_focus=true @AndreiNavoichyk

Comment (AndreiNavoichyk) [reply scripts/init.sh]: So we depend on `nightly` version of `wasm32-unknown-unknown`.
Ok, I've refactored a little bit the structure. Let's wait for build.

Comment (huythanhcere) [reply scripts/init.sh]: Thanks

File: .github/workflows/build-and-push-docker-image.yml [hunk 3]
```diff
@@ -2,8 +2,8 @@ name: Build and push image to ECR
 on:
   push:
     branches:
-      - dev-cere
-      
+      - 'dev-cere'
```
Comment (AndreiNavoichyk) [inline .github/workflows/build-and-push-docker-image.yml]: revert before merge

**Pull request merged**